include/open62541/client_highlevel.h

```diff
@@ -454,6 +464,16 @@ UA_Client_writeUserExecutableAttribute(UA_Client *client, const UA_NodeId nodeId
                                       &UA_TYPES[UA_TYPES_BOOLEAN]);
 }
 
+UA_StatusCode UA_EXPORT
+UA_Client_writeRolePermissionsAttribute(UA_Client *client, const UA_NodeId nodeId,
+                                       size_t newRolePermissionSize,
+                                       const UA_RolePermissionType *rolePermission);
+
+UA_StatusCode UA_EXPORT
```

Comment:
My guess is that the UserRolePermissions are auto-computed from the roles of the user and the permissions of these roles.
Similar to AccessRights vs UserAccessRights.
So this is not something that can be written, only read.
Check if `writeUserRolePermissionsAttribute` makes sense or should be removed.

Reply:
`writeUserRolePermissionsAttribute` is removed